data-structures-queues/postpone.js

```diff
@@ -0,0 +1,9 @@
+/* exported postpone */
+
+function postpone(queue) {
+  if (queue.peek() === undefined) {
+    return queue;
```

Comment:
`postpose` should not return anything:

---

![image](https://user-images.githubusercontent.com/16143771/156821039-1c54a760-52bc-437c-97ea-7f5353fc4c92.png)